src/bicep/mlz.bicep

```
@@ -252,6 +252,58 @@ module sharedServicesVirtualNetworkPeering './modules/spokeNetworkPeering.bicep'
   }
 }
 
+module hubPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
+  scope: resourceGroup(hubSubscriptionId, hubResourceGroupName)
+  dependsOn: [
+    hubResourceGroup
+    logAnalyticsWorkspace
+  ]
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroupName
+  }
+}
+
+module operationsPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: '${operationsResourceGroupName}-policyAssignment'
+  scope: resourceGroup(operationsSubscriptionId, operationsResourceGroupName)
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroup.name
+  }
+}
+
+module sharedServicesPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
+  scope: resourceGroup(sharedServicesSubscriptionId, sharedServicesResourceGroupName)
+  dependsOn: [
+    sharedServicesResourceGroup
+    logAnalyticsWorkspace
+  ]
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroupName
+  }
+}
+
+module identityPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
```

nit: from the deployment output I couldn't tell these apart from the other assignments applied at the same time

```suggestion
  name: 'policyAssignment-${identityResourceGroupName}'
```